docs/book/getting-started/deploying-zenml/helm.md

```diff
@@ -99,4 +99,31 @@ To disconnect from the current ZenML server and revert to using the local defaul
 
 ```bash
 zenml disconnect
-```
\ No newline at end of file
+```
+
+## Upgrading your ZenML server
+
+To upgrade your ZenML server Helm release to a new version, follow the steps below: 
+
+- Modify your `values.yaml` file that you used for the initial deployment to include the new image tag. The path to the tag is `zenml.image.tag`.
+- If you don’t have the previous `values.yaml` file handy, you can create a new one with your existing values using the following command:
+
+    ```bash
```

Comment:
Indent backwards

Reply:
![image](https://user-images.githubusercontent.com/37150991/213665279-be6872a0-c688-421f-aaa1-5bf6bd77b24d.png)
I have indented it in the current form so that it lines up with the bullet point indentation. Indenting it backwards would make the code block start from the very left. Lmk if you feel the latter looks better.

Reply:
It's also the same with all of our other code blocks in the doc.